docs/management/security/encryption.md

```diff
@@ -2,15 +2,15 @@
 title: "TLS"
 linkTitle: "TLS"
 weight: 1
-description: Redis TLS support
+description: Valkey TLS support
 aliases: [
     /topics/encryption,
     /docs/manual/security/encryption,
     /docs/manual/security/encryption.md
 ]
 ---
 
-SSL/TLS is supported by Redis starting with version 6 as an optional feature
+SSL/TLS is supported by Valkey starting with version 6 as an optional feature
```

Comment:
There is no Valkey version 6.

When refering to old redis version, write "Redis OSS". It is according to redis trademark policy.

Reply:
In this case we can just remove "starting with version 6". Imo

Reply:
done